.github/workflows/main.yml

```diff
@@ -30,7 +30,7 @@ jobs:
           node-version-file: ".nvmrc"
           cache: "pnpm"
       - name: Install dependencies
-        run: pnpm install --frozen-lockfile
+        run: pnpm install
```

Comment:
`--frozen-lockfile` is set by default in by pnpm when in CI env https://pnpm.io/cli/install#--frozen-lockfile